### PR TITLE
feat: #188 add test to dropdown

### PR DESCRIPTION
**Pull request by Alex89198900**
1. Add test to dropdown

### CODE REVIEW

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    cy.get(getDataCy(DROPDOWN_CY.dataCyContent))
+      .should("not.exist");
+  });
+
+  it("should render content", () => {
+    cy.get(getDataCy(DROPDOWN_CY.dataCyTrigger))
+      .click();
+    cy.get(getDataCy(DROPDOWN_CY.dataCyContent))
+      .should("exist");
+  });
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/dropdown/Dropdown.cy.tsx:66]: Also you can check
1. is options correct. (length and that each option render right value
2. Should select right option after click on value

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx (hunk 1 above)
Comment (Alex89198900) [reply mw-webapp/src/component/dropdown/Dropdown.cy.tsx:69]: Done

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  it("should length and that each option render right value", () => {
+    cy.get(getDataCy(DROPDOWN_CY.dataCyTrigger))
+      .click();
+    cy.get(getDataCy(DROPDOWN_CY.dataCyContentList)).each(($ele) => {
+      if ($ele.text() === "tes#1") {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/dropdown/Dropdown.cy.tsx:75]: Can you please explain what does $ele. mean?

Comment (tritonJS826) [reply mw-webapp/src/component/dropdown/Dropdown.cy.tsx:75]: for js it is not convenient to start name variables with "$" sign) just name it properly (for example "dropdownOption" etc.)

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        cy.wrap($ele).click();
+      }
+    });
+    cy.get(getDataCy(DROPDOWN_CY.dataCyContentList)).each(($ele) => {
+      if ($ele.text() === "tes#2") {
+        cy.wrap($ele).click();
+      }
```
Comment (tritonJS826) [inline mw-webapp/src/component/dropdown/Dropdown.cy.tsx:82]: you can get element by text in cypress

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    });
+    cy.get(getDataCy(DROPDOWN_CY.dataCyContentList)).find("li").should("have.length", DROPDOWN_LENGHTH);
+  });
+
+  it("should select right option after click on value", () => {
+    cy.get(getDataCy(DROPDOWN_CY.dataCyTrigger)).click();
+    cy.contains(getDataCy(DROPDOWN_CY.dataCyContentList), "tes#1").click();
+    cy.contains(getDataCy(DROPDOWN_CY.dataCyContentList), "tes#2").click();
```
Comment (tritonJS826) [inline mw-webapp/src/component/dropdown/Dropdown.cy.tsx:90]: if "tes" mean "test"  - lets use the full word

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx (hunk 4 above)
Comment (Alex89198900) [reply mw-webapp/src/component/dropdown/Dropdown.cy.tsx:90]: Done

File: mw-webapp/src/component/dropdown/Dropdown.cy.tsx (hunk 2 above)
Comment (Alex89198900) [reply mw-webapp/src/component/dropdown/Dropdown.cy.tsx]: Done

**Review by Ekaterina1994 — APPROVED** [review]

**Pull request merged**